sycl/plugins/xrt/CMakeLists.txt

```diff
@@ -0,0 +1,68 @@
+
+message(STATUS "Including the PI API Xilinx Runtime (XRT) backend.")
+
+
+if (NOT DEFINED XILINX_RT_INCLUDE)
+  # First check user environment
+  if (DEFINED ENV{XILINX_XRT} AND 
+      EXISTS "$ENV{XILINX_XRT}/include")
+      message(STATUS "Got XRT installation path from $XILINX_XRT")
+      set(XILINX_RT_INCLUDE "$ENV{XILINX_XRT}/include")
+      
+  # Check on standard XRT installation path
+  elseif(EXISTS "/opt/xilinx/xrt/include")
+    message(STATUS "Standard XRT installation path found")
+    set(XILINX_RT_INCLUDE "/opt/xilinx/xrt/include")
+  
+  #TODO: Automatic download of XRT once their CMake is clean
```

Comment:
Interesting feature. Do you mean a FETCH_CONTENT for example?
But let's not over-engineering it for now :-)